arch/risc-v/src/opensbi/Make.defs

```diff
@@ -30,6 +30,7 @@ OPENSBI_CSRCS += opensbi/opensbi-3rdparty/lib/utils/timer/aclint_mtimer.c
 
 OPENSBI_ASRCS += opensbi/opensbi-3rdparty/lib/sbi/sbi_expected_trap.S
 OPENSBI_ASRCS += opensbi/opensbi-3rdparty/lib/sbi/sbi_hfence.S
+OPENSBI_ASRCS += opensbi/opensbi-3rdparty/lib/sbi/riscv_hardfp.S
```

Comment:
```suggestion
ifeq ($(CONFIG_ARCH_FPU),y)
OPENSBI_ASRCS += opensbi/opensbi-3rdparty/lib/sbi/riscv_hardfp.S
endif
```

Reply:
But I'm not sure if this is needed

Reply:
let me check if `riscv_hardfp.S` is properly guarded

Reply:
yes, it is `#ifdef __riscv_flen`. So this is not needed

Reply:
Yep it was guarded so we can add the file safely, it will just become and empty object and garbage collection will wipe it